config/jobs/kubernetes-sigs/mcs-api/mcs-api-presubmits-master.yaml

```diff
@@ -74,6 +74,7 @@ presubmits:
           make test || result=$?
           cp cover.out ${ARTIFACTS}
           cd ../../k8s.io/test-infra/gopherage
+          GO111MODULE=on go build .
```

Comment:
I think when I was copying this over from CAPZ I had it in my head that this `go build` was running against the repo we're testing instead of the `gopherage` tool which isn't precompiled.